Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -1229,6 +1238,15 @@ void UpdateToolbarItems()
 					n.UpdateToolBarVisible();
 			}
 
+			void OnToolberItemPropertyChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
Typo in the method name

Reply:
@jfversluis 
Thank you for your comment.
I fixed it.

Reply:
@jfversluis 
Can I click the Resolve conversation button?

Reply:
Nope, because I did 😉  Thanks for resolving this so quickly!

Reply:
I see. Thanks!